concordium-std-derive/src/lib.rs

```diff
@@ -2121,6 +2125,66 @@ fn concordium_test_worker(_attr: TokenStream, item: TokenStream) -> syn::Result<
     Ok(test_fn.into())
 }
 
+#[cfg(feature = "concordium-quickcheck")]
+#[proc_macro_attribute]
+/// Derive the appropriate export for an annotated quickcheck function, when
+/// feature "wasm-test" is enabled, otherwise behaves like `#[quickcheck]`.
```

Comment:
It's a bit difficult to read from the code what the generated function actually is. Could you elaborate in the comment here what that does?

Reply:
Done